example_gh_test.go

```diff
@@ -153,6 +154,43 @@ func ExampleGQLClient_advanced() {
 	fmt.Println(query)
 }
 
+// Add a star to the cli/go-gh repository using the GQL API.
+func ExampleGQLClient_Mutate_simple() {
+	client, err := gh.GQLClient(nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var m struct {
+		AddStar struct {
+			Starrable struct {
+				Repository struct {
+					Stargazers struct {
+						Nodes []struct {
+							Login string
+						}
+					} `graphql:"stargazers(first: $first)"`
+				} `graphql:"... on Repository"`
+				Gist struct {
+					StargazerCount int
+					Url            string
+				} `graphql:"... on Gist"`
+			}
+		} `graphql:"addStar(input: $input)"`
+	}
+	variables := map[string]interface{}{
+		"input": githubv4.AddStarInput{
+			ClientMutationID: githubv4.NewString("addStar"),
```

Comment:
Let's remove `ClientMutationID` since it's not necessary for the request

Reply:
my intent with ClientMutationID was to show how to convert inputs, but it looks like that works for StarrableID as well 😄